Podcast/SearchSeriesTableViewCell.swift

```diff
@@ -81,15 +81,28 @@ class SearchSeriesTableViewCell: UITableViewCell {
     
     func configure(for series: Series, index: Int) {
         self.index = index
-        seriesImageView.image = #imageLiteral(resourceName: "filler_image")
+        if let url = series.smallArtworkImageURL {
+            if let data = try? Data(contentsOf: url) {
+                seriesImageView.image = UIImage(data: data)
+            }
+        } else {
+            seriesImageView.image = #imageLiteral(resourceName: "sample_series_artwork")
+        }
+        setSubscribeButtonToState(state: series.isSubscribed)
         titleLabel.text = series.title
         publisherLabel.text = series.author
         subscribersLabel.text = series.numberOfSubscribers.shortString() + " Subscribers"
     }
     
     func didPressSubscribeButton() {
-        subscribeButtonPressed = !subscribeButtonPressed
-        subscribeButton.isSelected = subscribeButtonPressed
-        delegate?.searchSeriesTableViewCell(cell: self, didSetSubscribeButton: subscribeButtonPressed)
+        delegate?.searchSeriesTableViewCellDidPressSubscribeButton(cell: self)
+    }
+    
+    func setSubscribeButtonToState(state: Bool) {
```

Comment:
instead of state, call the parameter isSelected: Bool. makes more sense when reading if isSelected {...}